env.sample

```diff
@@ -10,7 +10,7 @@ XDAI_RPC=https://xdai.poanetwork.dev
 AVAX_RPC=https://api.avax.network/ext/bc/C/rpc
 WAN_RPC=https://gwan-ssl.wandevs.org:56891
 HARMONY_RPC=https://api.s0.t.hmny.io
-THUNDERCORE_RPC=https://mainnet-rpc.thundercore.com
+THUNDERCORE_RPC=https://mainnet-rpc.thundercore.com/archived/DEFILLAMAaakTsRI6LnA
```

Comment:
I gave this a try and it did not work

Reply:
`Error: failed to meet quorum (method="getBlockNumber", params={}, results=[{"weight":1,"start":1681375441006,"error":{"reason":"bad response","code":"SERVER_ERROR","status":504,"headers":{"server":"awselb/2.0","date":"Thu, 13 Apr 2023 08:45:01 GMT","content-type":"text/html","content-length":"132","co...`

Reply:
How are you sending the request?

On our end, this is what we've tried
```
curl https://mainnet-rpc.thundercore.com/archived/DEFILLAMAaakTsRI6LnA \                                                           
  -X POST \
  -H "Content-Type: application/json" \
  --data '{"method":"eth_blockNumber","params":[],"id":1,"jsonrpc":"2.0"}'
```

And this is what we are getting
```
{"jsonrpc":"2.0","id":1,"result":"0x7ba0eac"}
```

lgtm

Reply:
also, this is not the right place for this. We need to either update in the SDK or set as env variable on our server.

Reply:
Ya sorry for the inconvenience mate.

Can you try the above curl and see if it works on your end? Also, can you send the request again, so I can have our devops debug the request. Cheers

Reply:
everytime I hit this endpoind, it fails, old endpoint works fine

Reply:
Hi team, I was told that we might have found and fixed the problem on the RPC. Can we try again? Cheers

Reply:
@peter-thundercore it works :rocket: 
the advantage of using the new rpc endpoint is, we get access to archival node and can query for older data?